docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1792,71 +1807,75 @@ or special exceptions that implicitly add `$document,subdocument`:
+
+#### 5. Specific exceptions: {#priority-category-4}
 
 [//]: # (Please keep them sorted)
 
 * [`$content`](#content-modifier),
 * [`$elemhide`](#elemhide-modifier),
 * [`$extension`](#extension-modifier),
+* [`$genericblock`](#genericblock-modifier),
+* [`$generichide`](#generichide-modifier),
 * [`$jsinject`](#jsinject-modifier),
 * [`$specifichide`](#specifichide-modifier),
-* [`$urlblock`](#urlblock-modifier),
-* [`$genericblock`](#genericblock-modifier),
-* [`$generichide`](#generichide-modifier);
+* [`$urlblock`](#urlblock-modifier);
 
-Each of which adds `10^3` to the priority.
+Each of which adds `10^4` to the priority.
 
-As well as
-- exception with a only one modifier (**without any other modifiers**) - [`$document`](#document-modifier) (because it's an alias for `$elemhide,content,jsinject,urlblock,extension`)
+As well as exception with [`$document modifier`](#document-modifier): because it's an alias for `$elemhide,content,jsinject,urlblock,extension`.
+It will add `10^4` for each modifier from [the top list](#priority-category-4), `10^4 * 5` in total.
 
-In addition, each of the exceptions implicitly adds the two allowed content-type modifiers `$document,subdocument`.
+In addition, each of these exceptions implicitly adds the two allowed content-type modifiers `$document,subdocument`.
 
-#### 5. Allowlist rule {#priority-category-5}
+#### 6. Allowlist rule {#priority-category-5}
 
-Modifier `@@` adds `10^4` to rule priority.
-
-#### 6. $redirect rules {#priority-category-6}
-
-The modifier [`$redirect`](#redirect-modifier) adds `10^5` to rule priority.
-Modifier [`$redirect-rule`](#redirect-rule-modifier) adds `10^5 + 1` to rule priority
-The `$redirect-rule` in the presence of a blocking rule is a higher priority than a normal `$redirect`.
+Modifier `@@` adds `10^5` to rule priority.
 
 #### 7. important rules {#priority-category-7}
 
 Modifier [`$important`](#important-modifier) adds `10^6` to rule priority.
 
 #### Rules for which there is no priority count {#priority-category-extra}
 
-[//]: # (Please keep them sorted)
-
-Rules that do not override blocking, but do additional post- or pre-processing of requests:
-* [`$cookie`](#cookie-modifier),
-* [`$csp`](#csp-modifier),
-* [`$jsonprune`](#jsonprune-modifier),
-* [`$removeheader`](#removeheader-modifier),
-* [`$removeparam`](#removeparam-modifier),
-* [`$replace`](#replace-modifier);
+[Other modifiers](#advanced-capabilities), which do not override a blocking but perform additional post- or pre-processing of requests, have no priority.
 
-Rules that do not need priority:
-* [`$badfilter`](#badfilter-modifier),
-* [`$stealth`](#stealth-modifier).
+> **Note**
+>
+> The [`$replace`](#replace-modifier) modifier takes precedence over all blocking rules of categories 1-3, as well as rules of exceptions from categories 3-5, **except** [`$content`](#content-modifier), because an exception with the `$content` modifier overrides all `$replace` rules.
```

Comment:
```suggestion
> The [`$replace`](#replace-modifier) modifier takes precedence over all blocking rules of categories 1-3, as well as exception rules from categories 3-5, **except** [`$content`](#content-modifier), because an exception with the `$content` modifier overrides all `$replace` rules.
```